sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/SampleDao.java

```diff
@@ -52,6 +52,6 @@
 
   Long countBySearch(String querystr) throws IOException;
 
-  Identity getIdentityByExternalName(String externalName) throws IOException;
+  Collection<Identity> getIdentitiesByExternalNameOrAlias(String externalName) throws IOException;
```

Comment:
Add Javadoc detailing parameter expectations and what exactly will be returned. Judging by the implementation in HibernateSampleDao, it looks like this is intended to find by a single external name (not comma-separated list), and return anything `LIKE '%externalName%'`. It would be easy to assume externalName should be a comma-separated list, since that's what `Identity#getExternalName()` returns